packages/cli/src/lib/commands/instance.ts

```diff
@@ -106,6 +106,26 @@ export const instance: CommandDefinition = (program) => {
                 { type: contentType, end });
         });
 
+    instanceCmd
+        .command("inout")
+        .argument("<id>", "Instance id or '-' for the last one started or selected")
+        .argument("[file]", "File with data")
+        .option("-t, --content-type <value>", "Content-Type", "text/plain")
```

Comment:
should use different API to add type validation (something like): .addOption(new Option(""-t, --content-type <value>", "Content-Type").choices("text/plain"))